pkg/pillar/types/dns.go

```diff
@@ -50,9 +50,11 @@ type NetworkPortStatus struct {
 	InvalidConfig        bool
 	Cost                 uint8
 	Dhcp                 DhcpType
-	Type                 NetworkType // IPv4 or IPv6 or Dual stack
-	Subnet               net.IPNet
```

Comment:
@rucoder Does Monitor use this `Subnet` field from `DeviceNetworkStatus`?
If yes, then I will need your help to switch it to using `IPv4Subnet/IPv6Subnets` instead.

Reply:
@milan-zededa yes, the monitor uses these fields. I can take a look what has to be done later next week. I can run tui on your pr and see where it crashes :)

Reply:
@milan-zededa yes, I came here to report the same. I'll have a look and try to fix it ASAP on top of your PR

Reply:
Thanks @rucoder!

Reply:
Depends what you want to display - the value configured by the user or the value actually applied (could differ if something failed)?

Reply:
OK, the bug is fixed, `IPv4Subnet` will be now properly set even in the static IP case.
@rucoder Please rebase your monitor changes on the latest published code to https://github.com/milan-zededa/eve/tree/ipv6-connectivity